src/additionalChecks.js

```diff
@@ -10,25 +10,29 @@ import {Job} from "./job.js";
 const validateCheck = function (checkConfig) {
     if (typeof checkConfig !== 'object' || checkConfig === null) {
         // NOT AN OBJECT!
-        core.warning("Skipping additional check; not an object, or is null: " + JSON.stringify(checkConfig));
+        core.warning(`Skipping additional check; not an object, or is null: ${  JSON.stringify(checkConfig)}`);
+
         return false;
     }
 
     if (checkConfig.name === undefined || checkConfig.job === undefined) {
         // Missing one or more required elements
-        core.warning("Skipping additional check due to missing name or job keys: " + JSON.stringify(checkConfig));
+        core.warning(`Skipping additional check due to missing name or job keys: ${  JSON.stringify(checkConfig)}`);
```

Comment:
Spacing got a bit weird here :thinking: 

Applies to other `${` conversions too.

Reply:
I've seen that. These were added by eslint `--fix`.
Gonna see if I can change this without eslint triggering errors.